test/contrib/test_pyopenssl.py

```diff
@@ -55,3 +57,15 @@ def test_dnsname_to_stdlib_leading_splat(self):
         expected_result = '*.xn--p1b6ci4b4b3a.xn--11b5bs8d'
 
         self.assertEqual(_dnsname_to_stdlib(name), expected_result)
+
+class TestPyOpenSSLErrorHandling(unittest.TestCase):
+    """
+    Tests for wrapping OpenSSL errors in urllib3 errors
+    """
+    def test_invalid_ca_certs(self):
+        """
+        Checks that we get an urllib3.exceptions.SSLError and not openssl.Error with missing ca_certs.
+        """
+        with self.assertRaises(SSLError):
+            http = urllib3.PoolManager(ca_certs='/no/path', cert_reqs='CERT_REQUIRED')
+            http.request('GET', 'https://python.org')
```

Comment:
This is a good effort, but it would be better if this could go in `test_https` and could attempt to contact the test server. It generally works a bit better to avoid the possibility of making outbound connections if at all possible.